aleks/resume.md

```diff
@@ -1,29 +1,46 @@
-## Résumé: Aleksandar Todorović 	
+# Résumé: Aleksandar Todorović 	
```

Comment:
😆 o.k. fine, h1 why don't you.

Reply:
I mean, it _is_ the start of the document, and as such deserves the headline. :smile: